decidim-admin/app/forms/decidim/admin/scope_form.rb

```diff
@@ -4,21 +4,29 @@ module Decidim
   module Admin
     # A form object to create or update scopes.
     class ScopeForm < Form
-      attribute :name, String
+      include TranslatableAttributes
+
+      translatable_attribute :name, String
       attribute :organization, Decidim::Organization
+      attribute :code, String
+      attribute :parent_id, Integer
+      attribute :scope_type_id, Integer
+      attribute :deprecated, Boolean
```

Comment:
This is probably a good idea, but I might be more comfortable discussing it in a separate feature. What do you think of shipping a MVP of this first (hierarchical scopes) and then refine it (deprecation, searchability, etc). What do you think? There's implications of this at the UI & UX level that I would feel more comfortable exploring better.